app/views/boats/show.html.erb

```diff
@@ -6,5 +7,5 @@
 <p> Price: <%= @boat.price_day %></p>
 <p> <%= cl_image_tag @boat.photo, height: 300, width: 400, crop: :fill %> </p>
 
```

Comment:
No option to go back from this view unless the user books it A bit harsh, I feel.